.gitignore

```diff
@@ -11,3 +11,15 @@ dist/
 
 # ci config for local ci build
 .circleci/local-config.yml
+
+# VS Code
+.vscode
+
+### Vim ###
+# Swap
+[._]*.s[a-v][a-z]
+!*.svg  # comment out if you don't need vector files
```

Comment:
Minor thing but can we move this out of the big chunk of `Vim` stuff?? Its not really related to Vim at all

Reply:
> Minor thing but can we move this out of the big chunk of `Vim` stuff?? Its not really related to Vim at all

Good point. How 'bout I just remove it. The `Masters of Vim` can correct if needed.  28781fb